cypress/support/pages/ContactUsPage.js

```diff
@@ -0,0 +1,59 @@
+var emailLocator = 'input[id="email"]';
```

Comment:
Nice to see the locators called out in variables rather than being in the implementation.